src/sass/686-dependent-status.scss

```diff
@@ -1,8 +1,8 @@
-@import "shared-variables";
+@import "~@department-of-veterans-affairs/jean-pants/sass/shared-variables";
 @import "modules/m-process-list";
```

Comment:
Why are some modules left out? Like `m-form-process` was moved into the design system, but not `m-process-list`.

Reply:
That particular one was a mistake, I meant to pull that one from jean-pants, too.

I think all of the common modules will get moved over unless they're specific to particular feature (like the downtime notification file, for example). I left the header/footer related ones in here for now because I'm a little fuzzy on the best way to organize that in the design system.